app/controllers/api/activities_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::ActivitiesController < ApplicationController
+  def create
+    Activity.transaction do
+      params[:activities].each do |activity|
+        @last_activity = Activity.new(activity_params(activity))
+        @last_activity.save!
+      end
+    end
+    render json: ['success!']
+
+  rescue ActiveRecord::RecordInvalid
```

Comment:
It is something that hound wants :( I'd disable it but I don't really have the time to bother but now that we have all of our configs in a centralized place we can deal with crazy linting stuff like this :)